helm-chart/values.yaml

```diff
@@ -1,61 +1,122 @@
+# Configs for the worker-tap daemonset
 tap:
-  docker:
+  image:
```

Comment:
The structure of `values.yaml` should match this Go struct:

https://github.com/kubeshark/kubeshark/blob/470ab3d7ed192a9c52b4b16f9638a037c36f16b2/config/configStruct.go#L29-L39

In fact, `values.yaml` shouldn't be edited manually but instead generated automatically using `make generate-helm-values` command:

https://github.com/kubeshark/kubeshark/blob/470ab3d7ed192a9c52b4b16f9638a037c36f16b2/Makefile#L72-L73